typescript/lambda-cloudwatch-dashboard/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "lambda-cloudwatch-dashboard",
+  "version": "0.1.0",
+  "bin": {
+    "lambda-cloudwatch-dashboard": "bin/lambda-cloudwatch-dashboard.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@aws-cdk/assert": "1.99.0",
+    "@types/jest": "^26.0.10",
+    "@types/node": "10.17.27",
+    "jest": "^26.4.2",
+    "ts-jest": "^26.2.0",
+    "aws-cdk": "1.99.0",
+    "ts-node": "^9.0.0",
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "@aws-cdk/core": "1.99.0",
+    "@aws-cdk/aws-lambda": "1.99.0",
+    "@aws-cdk/aws-cloudwatch": "1.99.0",
+    "source-map-support": "^0.5.16"
```

```suggestion
    "@aws-cdk/assert": "*",
    "@types/jest": "^26.0.10",
    "@types/node": "10.17.27",
    "jest": "^26.4.2",
    "ts-jest": "^26.2.0",
    "aws-cdk": "1.99.0",
    "ts-node": "^9.0.0",
    "typescript": "~3.9.7"
  },
  "dependencies": {
    "@aws-cdk/core": "*",
    "@aws-cdk/aws-lambda": "*",
    "@aws-cdk/aws-cloudwatch": "*",
    "source-map-support": "^0.5.16"
```

Tests are failing because the suggested changes are requierd.

Failure:
```sh
==============================
building project: typescript/lambda-cloudwatch-dashboard
==============================
=================================================
 These @aws-cdk dependencies must depend on version "*"
"@aws-cdk/assert": "1.99.0", "@aws-cdk/core": "1.99.0", "@aws-cdk/aws-lambda": "1.99.0", "@aws-cdk/aws-cloudwatch": "1.99.0",
=================================================
Error: Process completed with exit code 1.
```